platformsh-flex-env.php

```diff
@@ -228,3 +229,21 @@ function mapPlatformShMongoDatabase(string $relationshipName, Config $config): v
     setEnvVar('MONGODB_USERNAME', $credentials['username']);
     setEnvVar('MONGODB_PASSWORD', $credentials['password']);
 }
+
+function mapPlatformShRabbitMq(string $relationshipName, Config $config): void
+{
+    if (!$config->hasRelationship($relationshipName)) {
+        return;
+    }
+
+    $credentials = $config->credentials($relationshipName);
+
+    setEnvVar('MESSENGER_TRANSPORT_DSN', sprintf(
+        '%s://%s:%s@%s:%d/%%2f/messages',
+        $credentials['scheme'],
+        $credentials['username'],
+        $credentials['password'],
+        $credentials['host'],
+        $credentials['port']
```

Comment:
```suggestion
        $credentials['port'],
        $credentials['vhost'] ?? '%2f'
```

Reply:
There is no vhost parameter in the credentials array, is there?

Reply:
not on PE, but on Dedicated you have one ;-)